scripts/analyze

```diff
@@ -305,19 +305,22 @@ def write_jrsconfig(list):
                 feature = feature.replace(i, "#" + i)
             p.write(feature)
 
-def write_makefile(src, zconf, file):
-    """Write out a src Makefile"""
-    with open(args['MAKEBASE'], "r") as b:
+def write_cmakefile(src, zconf, file):
+    """Write out a src CMake file"""
+    with open(args['CMAKEBASE'], "r") as b:
         base = b.read()
     with open(file, "w") as f:
-        f.write(base)
+        f.write("# This is a generated file\n\n")
         if src:
-            f.write("obj-y += \\\n")
             for i in src:
-                f.write("\t" + i.replace(".c", ".o") + " \\\n")
-            f.write("\nccflags-y += \\\n")
+                f.write("target_sources(app PRIVATE " + i + ")\n")
+            f.write("\n")
             for i in zconf:
-                f.write("\t" + i + " \\\n")
+                """fixme: include paths should be a separate config"""
```

Comment:
Do you mean to fix this before merge?

Reply:
no, it's intended to be a todo in another patch, i'll file a issue to keep track of it